src/core/sim.py

```diff
@@ -50,10 +53,7 @@ def step(self) -> PropagatedOutput:
         return PropagatedOutput(self.state_time, self.observed_state)
 
     def should_stop(self) -> bool:
-        """Returns True if something in our state reaches a condition that should stop the sim
-
-        Args:
-            state (StateTime): The current state of the system to evaluate
+        """Returns true if something in our state reaches a condition that should stop the sim
```

Comment:
Kind of informal, probably just

Returns true if our state reaches a condition that should stop the sim